src/udiskslinuxblockobject.c

```diff
@@ -627,8 +627,8 @@ drive_does_not_detect_media_change (UDisksLinuxBlockObject *object)
   return ret;
 }
 
-static gboolean
-filesystem_check (UDisksObject *object)
+gboolean
+udisks_linux_block_object_filesystem_check (UDisksObject *object)
```

Comment:
Maybe we could also use this opportunity and give the function a better name?

Reply:
Changed to `udisks_linux_block_object_contains_filesystem()` and also added and fixed some doc strings.